### ISSUE TITLE
Use my own Stoploss value to calculate and remove ROI from Hyperopt

### ISSUE THREAD

**Issue by saikou999** [freqtrade/freqtrade#5932]
## Describe your environment

  * Operating system: MacOS
  * Python Version: python - 3.10.0
  * CCXT version: N/A
  * Freqtrade Version: freqtrade - 2021.10
  
## Your question

*Ask the question you have not been able to find an answer in the [Documentation](https://www.freqtrade.io/en/latest/)*

Hi,

I am trying to optmise my own strategy with my own stop loss. However when I run the following command, Hyperopt will always optimise the stop loss for me rather than my own stop loss. 

Here is how I defined my stop loss in my strategy. 

```
class MyStrategy(IStrategy):
    # Stoploss:
    stoploss = -0.017
```

`freqtrade hyperopt --hyperopt-loss SharpeHyperOptLoss --strategy MyStrategy --timerange 20200901-20211122 --spaces buy sell --eps`

Also, Hyperopt will always recommend me ROI, I don't want to use ROI in my strategy. How do I run hyperopt without it?
This is the result from Hyperopt

```
"params": {
    "roi": {
      "0": 0.087,
      "2": 0.024,
      "14": 0.015,
      "38": 0
    },
    "stoploss": {
      "stoploss": -0.28
    },
    "trailing": {
      "trailing_stop": false,
      "trailing_stop_positive": null,
      "trailing_stop_positive_offset": 0.0,
      "trailing_only_offset_is_reached": false
    },
    "buy": {
      "buy_max": 20,
      "buy_volitility": 0.047
    },
    "sell": {
      "sell_min": 67
    },
    "protection": {}
  }
```

Thank you.

**Comment by xmatthias** [comment]
Please get used to fill out the issue template. Repeat offenders to this will have their issues closed without further comment.
We're asking some very basic questions - by deleting the issue template, you're explicitly choosing not to answer our questions - which makes me doubt why you think we should chose to answer your questions.

---

If freqtrade loads a strategy and detects a parameter file (`StrategyName.json`) - it'll load all parameters from there (also for hyperopt - so you can move through spaces one by one).

Your problem is most likely that you initially ran an optimization with the stoploss space included (maybe `default`?), so now your parameters file has that -0.28 in it, and as you're never optimizing this again, it'll always "stay as is".

Please try changing this in the corresponding `.json` file to the desired value (or remove the parameter file if all other parameters are worthless to you) - then it should be the value that you want it to be (the one in the strategy actually has the least meaning, and will be overwritten by both the parameter file, as well as the configuration (should it be set there).
